pkg/peering/peering.go

```diff
@@ -263,6 +263,19 @@ func (m *Manager) ForAllConnected(f PeerConsumerFunc) {
 	}
 }
 
+// ForAll executes the given function for each peer until
+// abort is returned from within the consumer function.
+func (m *Manager) ForAll(f PeerConsumerFunc) {
+	m.RLock()
+	defer m.RUnlock()
+	for _, p := range m.connected {
+		if f(p) {
```

Comment:
Please flip the logic here and in `ForAllConnected` to abort on `false` instead of `true`.
This would match all other iterations in HORNET